docs/content/configuration/index.md

```diff
@@ -316,6 +316,7 @@ All processes that can serve queries can also log the query requests they see. B
 |Property|Description|Default|
 |--------|-----------|-------|
 |`druid.request.logging.type`|Choices: noop, file, emitter, slf4j, filtered, composing, switching. How to log every query request.|[required to configure request logging]|
+| `druid.request.logging.logSegmentMetadataQueries`| Boolean indicating if SegmentMetadata Queries should be logged by the Request Logger.| true|
```

Comment:
I think the way this property behaves is good, but it would be better inside FilteredRequestLogger. That way, it's composable, rather than needing to be implemented inside each and every RequestLogger.